tools/StaticAnalysis/Exceptions/Az.HDInsight/SignatureIssues.csv

```diff
@@ -68,3 +68,8 @@
 "Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll","Microsoft.Azure.Commands.HDInsight.ManagementCommands.AddAzureHDInsightClusterIdentity","Add-AzHDInsightClusterIdentity","1","8420","Parameter set 'CertificateFilePath' of cmdlet 'Add-AzHDInsightClusterIdentity' contains at least one parameter with a position larger than four, which is discouraged.","Limit the number of positional parameters in a single parameter set to four or fewer."
 "Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll","Microsoft.Azure.Commands.HDInsight.ManagementCommands.AddAzureHDInsightClusterIdentity","Add-AzHDInsightClusterIdentity","1","8420","Parameter set 'CertificateFileContents' of cmdlet 'Add-AzHDInsightClusterIdentity' contains at least one parameter with a position larger than four, which is discouraged.","Limit the number of positional parameters in a single parameter set to four or fewer."
 "Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll","Microsoft.Azure.Commands.HDInsight.NewAzureHDInsightClusterCommand","New-AzHDInsightCluster","1","8700","Parameter set '__AllParameterSets', 'CertificateFilePath', 'CertificateFileContents' of cmdlet 'New-AzHDInsightCluster' have the same mandatory parameters, and both of them are not default parameter set which may cause confusion.","Merge these parameter sets into one parameter set."
+"Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll","Microsoft.Azure.Commands.HDInsight.NewAzureHDInsightClusterAutoscaleScheduleConditionCommand","New-AzHDInsightClusterAutoscaleScheduleCondition","1","8100","New-AzHDInsightClusterAutoscaleScheduleCondition Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
```

Comment:
NewAzureHDInsightClusterAutoscaleScheduleConditionCommand
-  Please Implement should process 
-  Please change Days to Day

Reply:
> NewAzureHDInsightClusterAutoscaleScheduleConditionCommand
> 
> * Please Implement should process
> * Please change Days to Day

Hi @msJinLei , Thanks for your comments.  I have changed Days to Day.  But I think that we don't need to implement should process for NewAzureHDInsightClusterAutoscaleScheduleConditionCommand, because this cmdlet doesn't make change on the server side, it only creates an instance in local. How do you like?

Reply:
> > NewAzureHDInsightClusterAutoscaleScheduleConditionCommand
> > 
> > * Please Implement should process
> > * Please change Days to Day
> 
> Hi @msJinLei , Thanks for your comments. I have changed Days to Day. But I think that we don't need to implement should process for NewAzureHDInsightClusterAutoscaleScheduleConditionCommand, because this cmdlet doesn't make change on the server side, it only creates an instance in local. How do you like?

That's OK